tests/test_repo_utils.py

```diff
@@ -342,6 +347,47 @@ def test_tags_type_validation(self, tmpdir, tag, should_fail):
             # parsed result, it will be 1.14 without the double-quotes.
             assert [tag.replace('"', '')] == config.container['tags']
 
+    @pytest.mark.parametrize(
+        'files, error_msg',
+        (
+                (
+                        [REPO_CONTAINER_CONFIG_WRONG_EXTENSION],
+                        'Container config file has wrong extension please change it to '
+                        '{correct_file}'.format(correct_file=REPO_CONTAINER_CONFIG),
+                ),
+                (
+                        [REPO_CONTAINER_CONFIG, REPO_CONTAINER_CONFIG_WRONG_EXTENSION],
+                        'This repo contains both {correct_file} and {wrong_file}'.format(
+                            correct_file=REPO_CONTAINER_CONFIG,
+                            wrong_file=REPO_CONTAINER_CONFIG_WRONG_EXTENSION,
+                        ),
+                ),
+                (
+                        [REPO_CONTENT_SETS_FILE_WRONG_EXTENSION],
+                        'Content sets file has wrong extension please change it to '
+                        '{correct_file}'.format(correct_file=REPO_CONTENT_SETS_FILE),
+                ),
+                (
+                        [REPO_CONTENT_SETS_FILE, REPO_CONTENT_SETS_FILE_WRONG_EXTENSION],
+                        'This repo contains both {correct_file} and {wrong_file}'.format(
+                            correct_file=REPO_CONTENT_SETS_FILE,
+                            wrong_file=REPO_CONTENT_SETS_FILE_WRONG_EXTENSION,
+                        ),
+                ),
+                ([REPO_CONFIG_FILE], ''),
+                ([REPO_CONTENT_SETS_FILE], ''),
+        ),
+    )
+    def test_repo_files_extensions(self, tmpdir, files, error_msg):
+        for file in files:
```

Comment:
Don't shadow builtin `file`; maybe `repo_file`?

Reply:
noticed it, changed it, and then changed it again because I like your suggestion more.